src/AutoRest.CSharp/Mgmt/Generation/ResourceIdentifierChooser.cs

```diff
@@ -32,17 +30,5 @@ internal static Type GetResourceIdentifierType(this OperationGroup operation, Mg
         {
             return obj.Inherits != null ? obj.Inherits.Implementation as ObjectType : null;
         }
-
```

Comment:
I think GetObjectTypeBase can also be removed

Reply:
done